### PR TITLE
Add noteview in sidepanel

### LINKED ISSUE #98: View notes in side panel when view command is used

### PR DESCRIPTION
**Pull request by YuSoonZ**
Fix #98

### CODE REVIEW

**Review by kailashgautham — APPROVED** [review]
LGTM!

File: src/test/java/seedu/address/model/person/NotesTest.java [hunk 1]
```diff
@@ -61,7 +61,7 @@ public void isEmpty_nonEmptyNotes_returnsFalse() {
     public void toString_validNotes_correctStringRepresentation() {
         ArrayList<String> stringNotes = new ArrayList<>(Arrays.asList("Note 1", "Note 2", "Note 3"));
         Notes notes = new Notes(stringNotes);
-        assertEquals("Note 1\nNote 2\nNote 3\n", notes.toString());
+        assertEquals('\n' + "1: Note 1\n2: Note 2\n3: Note 3", notes.toString());
```
Comment (hiivan) [inline src/test/java/seedu/address/model/person/NotesTest.java:64]: Good catch!

**Review by hiivan — APPROVED** [review]
LGTM!

**Review by teojunda — APPROVED** [review]
LGTM！

**Pull request merged**